concepts/models.md

```diff
@@ -148,18 +148,124 @@ A field config is formatted as
 {% endtab %}
 {% endtabs %}
 
-where the &lt;field\_type&gt; is one of the following:
-
-| **Field type** | **Description** |
-| :--- | :--- |
-| `date` | A date string on the format "DD-MM-YYYY" |
-| `amount` | An amount string on the format "123.45" |
-| `alphanum` | Any string with alphanumeric format |
-| `alphanumext` | Any string with alphanumeric format, including some special characters |
-| `all` | Any string with alphanumeric format, including more special characters |
-| `letter` | Any string with latin characters only |
-| `number` | Any string with numbers only |
-| `phone` | Any phone number with or without country code |
+where the `<field_type>` is one of the following:
+
+<table>
+  <thead>
+    <tr>
+      <th style="text-align:left"><b>Field type</b>
+      </th>
+      <th style="text-align:left"><b>Data format</b>
+      </th>
+      <th style="text-align:left"><b>Characters in output</b>
+      </th>
+      <th style="text-align:left"><b>Output format</b>
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td style="text-align:left"><b><code>date</code></b>
+      </td>
+      <td style="text-align:left">Any valid date format</td>
+      <td style="text-align:left"><code>-0123456789</code>
+      </td>
+      <td style="text-align:left"><code>YYYY-MM-DD</code>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left"><b><code>amount</code></b>
+      </td>
+      <td style="text-align:left">Decimal number</td>
+      <td style="text-align:left"><code>-.0123456789</code>
+      </td>
+      <td style="text-align:left">
+        <p><code> 123.45</code>
+        </p>
+        <p><code>-123.45</code>
+        </p>
```

Comment:
Would it be more readable to put the closing </p> tags on the same line as the opening <p> tag?

Reply:
The closing tags are automatically formatted this way by gitbook, so I think it'll be futile to format these in the future, if we want to keep using the gitbook UI for writing.